Trappist network generation fails

```
papi add -w wss://rococo-trappist-rpc.polkadot.io/ trappist
papi generate
```

This fails with the following: 
```
Reading metadata
Generating code
Writing code
CLI Building entry: /Users/cornholio/work/parity/polkadot-testnet-faucet/node_modules/@polkadot-api/descriptors/src/index.ts
CLI Using tsconfig: tsconfig.json
CLI tsup v8.0.2
CLI Target: esnext
CJS Build start
ESM Build start
CJS You have emitDecoratorMetadata enabled but @swc/core was not installed, skipping swc plugin
ESM You have emitDecoratorMetadata enabled but @swc/core was not installed, skipping swc plugin
✘ [ERROR] Unexpected "#return"

    node_modules/@polkadot-api/descriptors/src/common-types.ts:1507:665:
      1507 │ ...>, input_per_byte: Anonymize<I4q39t5hn830vp>, r#return: Anonymize<I4q39t5hn830vp>, return_per_byte: A...
           ╵                                                   ~~~~~~~

✘ [ERROR] Unexpected "#return"

    node_modules/@polkadot-api/descriptors/src/common-types.ts:1507:665:
      1507 │ ...>, input_per_byte: Anonymize<I4q39t5hn830vp>, r#return: Anonymize<I4q39t5hn830vp>, return_per_byte: A...
           ╵                                                   ~~~~~~~

ESM Build failed
/Users/cornholio/work/parity/polkadot-testnet-faucet/node_modules/esbuild/lib/main.js:1651
  let error = new Error(text);
```

--- a/packages/client/CHANGELOG.md
+++ b/packages/client/CHANGELOG.md
@@ -9,6 +9,10 @@
 - Added subpath for `polkadot-api/chains/*`
 - Added subpath for `polkadot-api/utils`
 
+### Fixed
+
+- Codegen: generate struct types with property names that have special symbols.
+
 ## 0.1.0 - 2024-04-09
 
 ### Changed

--- a/packages/codegen/CHANGELOG.md
+++ b/packages/codegen/CHANGELOG.md
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Fixed
+
+- Generate struct types with property names that have special symbols.
+
 ## 0.0.1 - 2024-04-03
 
 ### Changed

--- a/packages/codegen/src/types-builder.ts
+++ b/packages/codegen/src/types-builder.ts
@@ -221,7 +221,7 @@ const _buildSyntax = (
     const deps = mapObject(value, buildNextSyntax)
     Object.values(deps).forEach(addImport)
     variable.type = `{${Object.entries(deps)
-      .map(([key, val]) => `${key}: ${anonymize(val.type)}`)
+      .map(([key, val]) => `${JSON.stringify(key)}: ${anonymize(val.type)}`)
       .join(", ")}}`
 
     return typesImport(name)

